fem/libceed/mass.cpp

```diff
@@ -11,52 +11,66 @@
 
 #include "mass.hpp"
 
-#include "ceed.hpp"
+#include "../../config/config.hpp"
 #ifdef MFEM_USE_CEED
-#include "mass.h"
+#include "mass_qf.h"
 #endif
 
 namespace mfem
 {
 
-void CeedPAMassAssemble(const FiniteElementSpace &fes,
-                        const mfem::IntegrationRule &irm, CeedData& ceedData)
+namespace ceed
+{
+
+#ifdef MFEM_USE_CEED
+struct MassOperatorInfo : public OperatorInfo
+{
+   MassContext ctx;
+   MassOperatorInfo()
+      : OperatorInfo{"/mass_qf.h",
+                     ":f_build_mass_const",
+                     ":f_build_mass_quad",
+                     ":f_apply_mass",
+                     ":f_apply_mass_mf_const",
+                     ":f_apply_mass_mf_quad",
+                     &f_build_mass_const,
+                     &f_build_mass_quad,
+                     &f_apply_mass,
+                     &f_apply_mass_mf_const,
+                     &f_apply_mass_mf_quad,
+                     EvalMode::Interp,
+                     EvalMode::Interp,
+                     1}
```

Comment:
My only concern with this is that it seems very easy to mix up the correct order of the parameters.

Reply:
An alternative is to write the following (I think it should work):
```c++
{
   header = "/mass_qf.h";
   build_func_const = ":f_build_mass_const";
   build_func_quad = ":f_build_mass_quad";
   apply_func = ":f_apply_mass";
   apply_func_mf_const = ":f_apply_mass_mf_const";
   apply_func_mf_quad = ":f_apply_mass_mf_quad";
   build_qf_const = &f_build_mass_const;
   build_qf_quad = &f_build_mass_quad;
   apply_qf = &f_apply_mass;
   apply_qf_mf_const = &f_apply_mass_mf_const;
   apply_qf_mf_quad = &f_apply_mass_mf_quad;
   trial_op = EvalMode::Interp;
   test_op = EvalMode::Interp;
   qdatasize = 1;
}
```

Reply:
I can also reorganize to pair together the corresponding `const char*` and `CeedQFunctionUser`, e.g. `":f_build_mass_const"` and `&f_build_mass_const`.

Reply:
I think the more explicit one is clearer, personally.